loklak.py

```diff
@@ -234,4 +234,45 @@ def main():
 	parser.add_argument('-u', '--user', action='store_true', help='User API to show twitter user information.')
 	parser.add_argument('-m', '--map', action='store_true', help='Map Visualization render using Loklak service.')
 	parser.add_argument('-md', '--markdown', action='store_true', help='Markdown conversion API to render markdown as image using Loklak.')
-	args = parser.parse_args()
+        if sys.version[0] == '2':
+            input = raw_input
+        args = parser.parse_args()
+        loklak = Loklak()
+        if args.search:
+            pprint(loklak.search(
+                 input('Enter query: '),
+                 input('Enter since: '),
+                 input('Enter until: '),
+                 input('Enter fromUser: ')
+            ))
+        elif args.status:
```

Comment:
Can we have a basic search system implemented without asking the user to take inputs. 
`loklak --search <query list to search for>`

This is a good attempt, but the aim is to limit the interaction and provide a single shot method to run a query. Can you modify this please ? The same goes for the user CLI